src/components/PrivateConsumerAccessFields.tsx

```diff
@@ -0,0 +1,308 @@
+import React, { FC, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Course, ResourceReadAccess, ResourceReadAccessNames } from '../types/resourceReadAccess.types';
+import styled from 'styled-components';
+import { Chip, CircularProgress, List, ListItem, ListItemText, Typography } from '@material-ui/core';
+import Button from '@material-ui/core/Button';
+import ErrorBanner from './ErrorBanner';
+import AddIcon from '@material-ui/icons/Add';
+import Popover from '@material-ui/core/Popover';
+import {
+  deleteAdditionalUserConsumerAccess,
+  deleteCourseConsumerAccess,
+  deleteCurrentUserInstitutionConsumerAccess,
+  getCoursesForInstitution,
+  getResourceReaders,
+  postCurrentUserInstitutionConsumerAccess,
+} from '../api/sharingApi';
+import { useSelector } from 'react-redux';
+import { RootState } from '../state/rootReducer';
+import { useFormikContext } from 'formik';
+import { Resource } from '../types/resource.types';
+import { Colors } from '../themes/mainTheme';
+import CancelIcon from '@material-ui/icons/Cancel';
+import PrivateConsumerCourseAccessFields from './PrivateConsumerCourseAccessFields';
+import PrivateConsumerPersonalAccessFields from './PrivateConsumerPersonalAccessFields';
+
+const StyledPrivateAccessFields = styled.div`
+  margin-top: 2.5rem;
+`;
+
+const StyledChipWrapper = styled.div`
+  display: block;
+`;
+
+const StyledChip = styled(Chip)`
+  && {
+    margin-top: 1rem;
+    margin-right: 0.5rem;
+    height: 100%;
+    color: ${Colors.Background};
```

Comment:
lurer på om vi burde satt font-weight til bold når vi har lys tekst på mørk bakgrunn for bedre lesbarhet

Reply:
Fikset